README.md

```diff
@@ -37,12 +28,10 @@ yetAnotherLocale =
 formatFloat : Locale -> Float -> String
 ```
 
-Format a `Float` number as a pretty string. For example, using the locales defined above:
+Format a `Float` number as a pretty string. For example, using the locale defined above:
 
 ```elm
 formatFloat defaultLocale 1234.5567 == "1,234.56"
-formatFloat alternativeLocale -7654.3210 == "-7.654,321"
```

Comment:
Ok, that's really good you added a lot of edge cases, I mean it ❤️ 

But having it _half-documented_ in the `README.md` and fully documented elsewhere leads to confusion. I'm OK with:

1. having it all on the `README.md` and fully duplicated at Elm Packages
1. **or** having everything at Elm Packages (and `README.md` just points there, as you suggested in this PR).

But anywhere in-between leads to confusion IMHO. Let's go for option number 2?